docs/source/management/monitoring/setup.md

```diff
@@ -42,12 +120,10 @@ Scylla Operator will notice the new ScyllaDBMonitoring object, and it will recon
 
 ### Wait for ScyllaDBMonitoring to roll out
 ```console
-kubectl wait --for='condition=Progressing=False' scylladbmonitorings.scylla.scylladb.com/example
-kubectl wait --for='condition=Degraded=False' scylladbmonitorings.scylla.scylladb.com/example
 kubectl wait --for='condition=Available=True' scylladbmonitorings.scylla.scylladb.com/example
 ```
 
-### Wait for Prometheus to roll out
```

Comment:
As discussed on a call, this is required because ScyllaDBMonitoring controller doesn't wait for the underlying infra to be actually available. Added an explicit note on it.